projects/components/src/menu-dropdown/menu-item/menu-item.component.test.ts

```diff
@@ -0,0 +1,24 @@
+import { fakeAsync } from '@angular/core/testing';
+import { IconType } from '@hypertrace/assets-library';
+import { Color } from '@hypertrace/common';
+import { MenuItemComponent } from '@hypertrace/components';
+import { createHostFactory } from '@ngneat/spectator/jest';
+
+describe('Menu Item Component', () => {
+  const createHost = createHostFactory<MenuItemComponent>({
+    component: MenuItemComponent,
+    shallow: true
+  });
+
+  test('should display icon and label as expected', fakeAsync(() => {
+    const spectator = createHost(
+      '<ht-menu-item [icon]="icon" [label]="label" [iconColor]="iconColor"></ht-menu-item>',
+      {
+        hostProps: { icon: IconType.MoreHorizontal, label: 'Item', iconColor: Color.Gray1 }
+      }
+    );
+    expect(spectator.query('.menu-item', { root: true })).toExist();
```

Comment:
why are these root queries? Also should we verify the icon and icon color beyond existence? Mocking the icon component would make that trivial

Reply:
oops. thats left over from a copy paste. removing.

I didnt' want to check the exact icon and color because that is tested in the icon component. Thought it'd be redundant.

Reply:
Not checking the resolution of them but the input - so checking that we correctly wire the icon and color inputs from this component to the icon component. In otherwords, something like `expect(spectator.query(IconComponent)?.icon).toBe(IconType.MoreHorizontal);`

Reply:
Importing the module is still using the real one - this is just bringing in all the real icon dependencies too. See below diff, which works with no test changes:
```diff
diff --git a/projects/components/src/menu-dropdown/menu-item/menu-item.component.test.ts b/projects/components/src/menu-dropdown/menu-item/menu-item.component.test.ts
index 33defb3..8681630 100644
--- a/projects/components/src/menu-dropdown/menu-item/menu-item.component.test.ts
+++ b/projects/components/src/menu-dropdown/menu-item/menu-item.component.test.ts
@@ -1,14 +1,15 @@
 import { fakeAsync } from '@angular/core/testing';
-import { IconLibraryTestingModule, IconType } from '@hypertrace/assets-library';
+import { IconType } from '@hypertrace/assets-library';
 import { Color, NavigationService } from '@hypertrace/common';
-import { IconComponent, IconModule, MenuItemComponent } from '@hypertrace/components';
+import { IconComponent, MenuItemComponent } from '@hypertrace/components';
 import { createHostFactory, mockProvider } from '@ngneat/spectator/jest';
+import { MockComponent } from 'ng-mocks';
 
 describe('Menu Item Component', () => {
   const createHost = createHostFactory<MenuItemComponent>({
     component: MenuItemComponent,
-    imports: [IconModule, IconLibraryTestingModule],
     shallow: true,
+    declarations: [MockComponent(IconComponent)],
     providers: [mockProvider(NavigationService)]
   });
```